### ISSUE TITLE
Two problems

### ISSUE THREAD

**Issue by ThorvaldAagaard** [EdwardPiwowar/BBA#580]
05A2C90C7DC5021D5191AE63EB3E
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/b9d294d3-586a-4c0e-890c-87a4317d0564)

In room 2 the south hand is not strong enough to a jump to 3H, yes 13 and a singleton, but CQ is not worth 2 HCP

In room 1 NS misses the 4-4 fit in Hearts, so I assume 2D is wrong especially as it denies 4-card major
After the invitational 2N South should probably pass, but 3N could be right,

**Comment by ThePokerDude** [comment]
There is a fundamental problem with negative doubles.
The description of 2d says: 3 or less hearts but we have four.
![image](https://github.com/user-attachments/assets/5fda2909-2387-4dc2-8365-50efab970cf3)

**Comment by ThorvaldAagaard** [comment]
Yes, the right bid is 2Ma if a 4-card, unless a good 6-card diamonds, that dislike a 4-3 fit

**Comment by EdwardPiwowar** [comment]
Next BBA (was old bug)
![image](https://github.com/user-attachments/assets/38970724-24d1-4b64-9f0b-1078ee6f1a23)
